front-end/src/components/navigation-icon/NavigationIcon.scss

```diff
@@ -0,0 +1,17 @@
+.navigationIcon {
+    text-align: center;
+    font-size: 12px;
+    font-family: Calibri;
```

Comment:
Similarly, use font-family on a reusable component or define globally. We dont want to define font family again and again and again. The instant we forgot this it will cause bugs

Reply:
Agreed. Fonts are defined (with their resources imported from the Cultural Events project) in an upcoming PR, so it should be taken care of then. This is a temporary value for the font.